plugins/dell/dell.quirk

```diff
@@ -0,0 +1,8 @@
+[fwupd-plugin-uefi-version-format]
```

Comment:
Should this be `FU_QUIRKS_PLUGIN_UEFI_VERSION_FORMAT` for consistency?

Reply:
Well, the #define has to start with `FU_QUIRKS` to make the gtk-doc work. I guess we could have `[fu-quirks-plugin-uefi-version-format]` if you think that's clearer?

Reply:
I was just meaning for consistency that when you search the project you find all the instances of the #define string in the same place.  So use them in the .quirk file rather than what they turn into.

Reply:
I think doing that makes it very complicated indeed. We'd need to have a table of key-values and also generate the `fu-quirks.h` file too... :(